asv_bench/benchmarks/benchmarks.py

```diff
@@ -376,6 +403,40 @@ def time_head(self, shape, head_count):
         execute(self.df.head(self.head_count))
 
 
+class TimeTail:
+    param_names = ["shape", "tail_count"]
+    params = [
+        get_benchmark_shapes("TimeTail"),
+        [5, 0.8],
```

Comment:
Is it worth adding other values for `tail_count` (and `head_count` for TimeHead) here? This could be a continuous range of percentages of the df to get head/tail for, e.g. `np.arange(0.1, 1, 0.1)`.

Reply:
It seems to me that this change does not cover any new use case for the functions.